### ISSUE TITLE
Domiciliation

### ISSUE THREAD

**Issue by webaaz** [francejs/association#5]
Pensez à une boite genre abcliv.fr pour fixer la domiciliation et ne pas la changer tous les ans si changement de trésorier (cela implique une modification des statuts)

**Comment by t8g** [comment]
Yes merci pour l'astuce :)

**Comment by MathRobin** [comment]
Cette domiciliation me semble nécessaire par contre, il ne faut pas oublier qu'elle a un coût. Pas énorme, mais à prévoir.

**Comment by nfroidure** [comment]
Oui, ça dépend du coût. Autre problème, si il y a augmentation du coût, on est liés sauf si ça coûte moins cher de modifier les statuts.
